README.md

```diff
@@ -137,8 +138,30 @@ Removing a version from the build won't make it be removed from Docker registry,
 
 ## Using and extending
 
+### PHP FPM healthcheck
+
+This image ships with the [php-fpm-healthcheck](https://github.com/renatomefi/php-fpm-healthcheck) which allows you to healthcheck FPM apart from the Nginx setup, providing more compatibility with the single process Docker container.
+
+This healthcheck provides diverse metrics to be watched upon and can be fitted accordingly to your needs, more information in how to use it in the [official documentation](https://github.com/renatomefi/php-fpm-healthcheck#a-php-fpm-health-check-script).
```

Comment:
```suggestion
This healthcheck provides diverse metrics to watch and can be configured according to your needs. More information on how to use it can be found in the [official documentation](https://github.com/renatomefi/php-fpm-healthcheck#a-php-fpm-health-check-script).
```